build-image/webif.bash

```diff
@@ -45,12 +45,12 @@ if [ "$1" = "inst_done" ]; then
   # shellcheck disable=SC2016
   echo '<html>
 <head>
-<meta http-equiv='"refresh"' content='"10;url=http://${HOSTNAME:-$hostname}:8080/"' />
+<meta http-equiv='"refresh"' content='"120;url=http://${HOSTNAME:-openhab}:8080/"' />
```

Comment:
is 120 the right number ? Ok for a start.
On a sidenote, the install log is refreshed every 10 secs, but it's always jumping back to the top of file.
Do you know how to jump to the bottom instead ?

Reply:
No clue, it was just a random value I chose. If it is not enough and we observe it often then we can change it.

No, I looked into it a week or two ago but didn't find anything.

Reply:
http://sourcetricks.com/2010/07/javascript-scroll-to-bottom-of-page.html
but it doesn't work :(